lib/screen/feralfile_series/feralfile_series_page.dart

```diff
@@ -27,13 +30,42 @@ class FeralFileSeriesPage extends StatefulWidget {
 class _FeralFileSeriesPageState extends State<FeralFileSeriesPage> {
   late final FeralFileSeriesBloc _feralFileSeriesBloc;
   final _canvasDeviceBloc = injector.get<CanvasDeviceBloc>();
+  final PagingController<int, Artwork> _pagingController =
+      PagingController(firstPageKey: 0);
+  static const _pageSize = 300;
 
   @override
   void initState() {
     super.initState();
     _feralFileSeriesBloc = context.read<FeralFileSeriesBloc>();
     _feralFileSeriesBloc.add(FeralFileSeriesGetSeriesEvent(
         widget.payload.seriesId, widget.payload.exhibitionId));
+    _pagingController.addPageRequestListener((pageKey) async {
+      await _fetchPage(pageKey);
+    });
+  }
+
+  Future<void> _fetchPage(int pageKey) async {
```

Comment:
_fetchArtworks

Reply:
It's fine/common to use fetchPage for this listener of infinite paging

Reply:
It's okay, just my felling, I don't know what this function do when read the function name. IMO, `fetchArtworks` is easier to understand.